BFWControls/Modules/Table/Controller/StaticTableViewController.swift

```diff
@@ -126,6 +128,7 @@ class StaticTableViewController: UITableViewController {
         if intrinsicHeightCells || filledUsingLastCell {
             tableView.estimatedRowHeight = 44.0
         }
+		NotificationCenter.default.addObserver(self, selector: #selector(StaticTableViewController.UIApplicationDidChangeStatusBarFrameHandler(for:)), name: NSNotification.Name.UIApplicationDidChangeStatusBarFrame, object: nil)
```

Comment:
- Added ```removeObserver``` from deinit for iOS 8 or earlier in latest commit
- The reason I didn't do it in ```viewDidLayoutSubviews()``` is that it also triggered by scrolling as well which feels like too much overhead for performance.